### DISCUSSION TITLE
Add support for websockets

### DISCUSSION THREAD

**Discussion by clayrisser** [onejs/one#616] [Ideas]
I am trying to create a graphql endpoint using this framework, but I need websockets to work for graphql subscriptions.

**Comment by clayrisser** [comment]
Solving the following issue would resolve this.

https://github.com/onejs/one/issues/191

**Comment by prabirshrestha** [comment]
+1 would love to have websocket support without creating yet another server.

**Comment by natew** [comment]
Agree this would be nice, only issue is we run Vite Connect server in dev, Hono in prod, so we'd need to somehow make that work. I guess just a onDevServer and onProdServer so people can implement themselves.
